HW13/proxy/proxy_txt_writer.py

```diff
@@ -0,0 +1,21 @@
+from HW13.proxy.txt_writer import TxtWriter
+
+
+class TxtProxyWriter:
+    def __init__(self, file_path):
+        self.__result = None
+        self.__is_actual = False
+        self.__txt_writer = TxtWriter(file_path)
+
+    def write_file(self):
+        if not self.__is_actual:
+            self.__result = self.__txt_writer.write('Hello')
+            self.__is_actual = True
+        return self.__result
```

Comment:
so in case I want to write new data but my file in some actual state it will not write it?